requirements.txt

```diff
@@ -9,5 +9,5 @@ semantic_version == 2.8.5
 six>=1.11.0,<1.15.0
```

Comment:
can we maybe also bump `six` constraint as well?

seeing some issue in https://github.com/Homebrew/homebrew-core/pull/136977

```
pkg_resources.DistributionNotFound: The 'six<1.15.0,>=1.11.0' distribution was not found and is required by awsebcli
```

Reply:
happy to file a different PR on it.

Reply:
@chenrui333 I think that maybe another PR would be better

Reply:
Totally, I will do a different PR then.